RelhaxModpack/RelhaxModpack/Utils/Translations.cs

```diff
@@ -153,6 +166,9 @@ public static void SetLanguageOnFirstLoad()
                 case "fr":
                     SetLanguage(Languages.French);
                     break;
+                case "ru":
```

Comment:
this needs to be changed to `es` see:
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.displayname?view=netframework-4.8